scripts/updateapilist.py

```diff
@@ -57,8 +58,14 @@ class MissingGithubToken(ValueError):
 # BASE_API defines the base API for Github.
 BASE_API = "https://api.github.com"
 
+# GITHUB_ISSUES defines the issues url for a repository on Github.
+GITHUB_ISSUES = "https://github.com/{repo}/issues"
 
+# BASE_ISSUE_TRACKER defines the base url for issue tracker.
+BASE_ISSUE_TRACKER = "https://issuetracker.google.com"
 
+# GENERIC_ISSUE_TRACKER_COMPONENT defines a generic component for issue tracker.
```

Comment:
This comment will leave us/others puzzled in the future. Let's briefly expand why we care:

"This issue-tracker component is part of some saved searches for listing API-side issues. However, when we construct URLs for filing new issues (which in some cases we do by analyzing the query string for a saved search), we want to ensure we DON'T file a new issue against this generic component but against a more specific one."

Reply:
addressed.